src/pages/workspace/qbo/PolicyQuickbooksImportPage.tsx

```diff
@@ -0,0 +1,126 @@
+                case NAVIGATION_KEYS.CUSTOMERS:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_CUSTOMER.getRoute(policyID));
+                    break;
+                case NAVIGATION_KEYS.LOCATIONS:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_LOCATIONS.getRoute(policyID));
+                    break;
+                case NAVIGATION_KEYS.TAXES:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_TAXES.getRoute(policyID));
+                    break;
+                default:
+                    break;
+            }
+        },
+        [policy?.id],
+    );
+
+    const sections = [
+        {
+            description: translate('workspace.qbo.accounts'),
+            navigationKey: NAVIGATION_KEYS.ACCOUNTS,
+            errorIndicator: !!policy?.errors?.syncAccounts,
+            title: syncAccounts,
+        },
+        {
+            description: translate('workspace.qbo.classes'),
+            navigationKey: NAVIGATION_KEYS.CLASSES,
+            errorIndicator: !!policy?.errors?.syncClasses,
+            title: syncClasses,
+        },
+        {
+            description: translate('workspace.qbo.customers'),
+            navigationKey: NAVIGATION_KEYS.CUSTOMERS,
+            errorIndicator: !!policy?.errors?.syncCustomers,
+            title: syncCustomers,
+        },
+        {
+            description: translate('workspace.qbo.locations'),
+            navigationKey: NAVIGATION_KEYS.LOCATIONS,
+            errorIndicator: !!policy?.errors?.syncLocations,
+            title: syncLocations,
+        },
+        {
+            description: translate('workspace.qbo.taxes'),
+            navigationKey: NAVIGATION_KEYS.TAXES,
+            errorIndicator: !!policy?.errors?.syncTaxes,
+            title: syncTaxes,
+        },
+    ];
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            shouldEnableMaxHeight
+            testID={PolicyQuickbooksImportPage.displayName}
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.qbo.import')}
+                onBackButtonPress={Navigation.goBack}
+            />
+            <ScrollView contentContainerStyle={styles.pb2}>
+                <Text style={[styles.pl5, styles.pb5]}>{translate('workspace.qbo.importDescription')}</Text>
+                {sections.map((section) => (
+                    <OfflineWithFeedback>
+                        <MenuItemWithTopDescription
+                            title={quickbooksOnlineConfigTitles[`${section.title ?? 'NONE'}`]}
+                            description={section.description}
+                            shouldShowRightIcon
+                            onPress={() => onPressConfigOption(section.navigationKey)}
+                            brickRoadIndicator={section.errorIndicator ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
```

Comment:
```suggestion
                            brickRoadIndicator={section.errorIndicator && CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR}
```

Reply:
Based on TS for brickRoadIndicator - it can't be boolean - either 'error' or undefined
TS2322: Type false | "error" is not assignable to type